3_Risk.md

```diff
@@ -7,6 +7,7 @@ Qualified Committers | Contributions over time and what components they commit t
 User Dependency | Number of users who are aware that they depend on the software over time
 Paid developers | Number of paid developers in community over time

```

Comment:
Paid by who?  There are a lot of developers who are paid by her/his employer to contribute to an open source project (e.g. OpenStack, Kubernetes, etc.)

Reply:
Paid to work on the project vs. paid but not specifically to work on the project. - I don't know how to best differentiate the two.